cmd/osm-controller/osm-controller.go

```diff
@@ -58,6 +58,8 @@ var (
 	caBundleSecretName string
 	osmConfigMapName   string
 
+	maxEnvoyConnect int
```

Comment:
The argument name exposes the implementation detail of using Envoy. If we change sidecar in the future, the argument name needs to be changed. Shall we use `maxDataPlaneConnection` or something else?

WDYT?

Reply:
That's a good point, I'll go ahead and make the change!